modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectStops.java

```diff
@@ -199,7 +199,7 @@ boolean containsStops(JSONObject jsonObject) {
             String key = (String) keys.next();
             if (jsonObject.get(key) instanceof JSONObject) {
                 JSONObject child = (JSONObject) jsonObject.get(key);
-                if (child.containsKey("stops")) {
+                if (child.containsKey("stops") && ((JSONArray)((JSONArray)child.get("stops")).get(0)).get(0) instanceof JSONObject) {
```

Comment:
This looks brittle, and possibly wrong.

Taking one of the examples below
```
"stops": [
             [10000, "red"],
             [100000, "blue"],
             [1000000, "green"],
             [1000000000, "black"]
           ]
```

stops.get(0) = [10000, "red"]
stops.get(0).get(0) = 10000, which should be a number, not a JSONObject.

If you did want to do more checks, you should probably do some `instanceof JSONArray` checks too, rather than just blindly casting.